consent/handler.go

```diff
@@ -430,13 +436,18 @@ func (h *Handler) RejectLoginRequest(w http.ResponseWriter, r *http.Request, ps
 //     Responses:
 //       200: consentRequest
 //       401: genericError
+//       409: genericError
 //       500: genericError
 func (h *Handler) GetConsentRequest(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	request, err := h.M.GetConsentRequest(r.Context(), ps.ByName("challenge"))
 	if err != nil {
 		h.H.WriteError(w, r, err)
 		return
 	}
+	if request.WasHandled {
+		h.H.WriteError(w, r, pkg.ErrConflict)
```

Comment:
I think it would be better to have a descriptive error here like the one from the manager: `fosite.ErrInvalidRequest.WithDebug("Consent verifier has been used already")`.